.github/workflows/build.yaml

```diff
@@ -59,9 +57,11 @@ jobs:
               with:
                   php-version: ${{ matrix.php }}
             - run: |
-                  sed -ri 's/"symfony\/(.+)": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
+                  sed -ri 's/"symfony\/(expression-language|framework-bundle|templating)": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
               if: matrix.symfony
             - run: composer config minimum-stability dev
               if: matrix.dev
             - run: composer update --no-interaction --no-progress --ansi ${{ matrix.composer_option }}
             - run: vendor/bin/phpunit
+              env:
+                  SYMFONY_DEPRECATIONS_HELPER: max[self]=0
```

Comment:
This should be reverted. For other jobs that the `Lowest deps` one, we still want deprecations coming from vendors to fail the tests (outside legacy tests). The whole point of using the bridge is that our CI fail when we use deprecated APIs instead of discovering them only when our users complain they get deprecations in their projects.

Reply:
reverted